assets/js/googlesitekit-dashboard.js

```diff
@@ -42,17 +42,15 @@ class GoogleSitekitDashboard extends Component {
 
 		if ( showModuleSetupWizard ) {
 			return (
-				<ErrorHandler>
-					<Setup />
-				</ErrorHandler>
+				<Setup />
```

Comment:
If it's simply returning `<Setup />` here then the parentheses can be removed and this can be just a single line 👍